docs/general/ecosystem-funds.md

```diff
@@ -0,0 +1,84 @@
+---
+id: ecosystem-funds
+title: Polkadot Ecosystem Development Funds
+sidebar_label: Ecosystem Funds
+description: An Overview of Trusted Polkadot Ecosystem Development Funds.
+keywords: [funding, vc, venture, venture capital, fund, ecosystem fund]
+slug: ../ecosystem-funds
+---
+
+The Polkadot has Ecosystem Development Funds aiming to support the development of rollups and
```

Comment:
parachains -> rollups 

This renaming is still in the discussion phase. If your intention is to introduce them to a brand new audience and remove the ambiguity, it would be better to say Parachains (Polkadot rollups). Rollups can also be built with Polkadot SDK but have nothing to do with the Polkadot ecosystem. Starknet, Avail are some examples.

Reply:
changed for now